### ISSUE TITLE
No utility in overriding help/version given exception

### ISSUE THREAD

**Issue by voltechs** [ManageIQ/trollop#87]
Due to the handling of `:help`/`:version` flags by raising an exception, there is little to no utility in overriding help. I suggest not raising an exception (arguably there is no exception) or providing a more comprehensive way to override this behavior if a user/developer supplies his/her own `:help`/`:version` options.

**Comment by nanobowers** [comment]
Agreed.  It seems one would need to override `educate` to fully reformat the help message.  Would think this could be implemented in a more flexible manner.
